test/api/controllers/bulkController.test.js

```diff
@@ -1,79 +1,71 @@
 var
-  should = require('should'),
-  params = require('rc')('kuzzle'),
-  Kuzzle = require.main.require('lib/api/Kuzzle'),
   Promise = require('bluebird'),
+  should = require('should'),
+  sinon = require('sinon'),
+  BulkController = require('../../../lib/api/controllers/bulkController'),
```

Comment:
I find it more elegant, within tests files, to require files from the app root, instead of those ugly relative paths (if we move the test file, it will be broken)